external_api/api.py

```diff
@@ -10,8 +10,10 @@
     FetchIndexForAliasException, DeleteIndexFromAliasException
 from chatbot_ner.config import ner_logger
 from external_api.constants import ENTITY_DATA, ENTITY_NAME, LANGUAGE_SCRIPT, ENTITY_LIST, \
-    EXTERNAL_API_DATA, SENTENCE_LIST
+    EXTERNAL_API_DATA, SENTENCE_LIST, CLOUD_STORAGE, ES_CONFIG, CLOUD_EMBEDDINGS, LIVE_CRF_MODEL_PATH
```

Comment:
rename this `CLOUD_STORAGE` and `CLOUD_EMBEDDINGS`

Reply:
done

Reply:
done